airflow/providers/qubole/example_dags/example_qubole.py

```diff
@@ -64,7 +64,7 @@ def compare_result_fn(**kwargs):
         :return: True if the files are the same, False otherwise.
         :rtype: bool
         """
-        ti = kwargs['ti']
+        ti = get_current_context()["ti"]
```

Comment:
```suggestion
```

Reply:
Updated to pass in the task instance object and remove the get_current_context() function call.